.travis.yml

```diff
@@ -115,6 +115,12 @@ script:
     if [[ -v BUILD_PRESTO_DOCKER ]]; then
       docker/build-local.sh
     fi
+  - |
+    if [[ -v MAVEN_CHECKS ]]; then
+      # TODO enable this for presto-parser
+      env MAVEN_OPTS="${MAVEN_INSTALL_OPTS}" ./mvnw clean compile test-compile -DskipTests -B -T C1 -P errorprone-compiler-presto \
```

Comment:
maybe extract this as a script (./bin/errorprone_compilation.sh) so it could be easier to reproduce issues locally.

Reply:
this would need to be parameterized with project list to be useful locally
i don't think having a script for this is useful